docs/reset-password.md

```diff
@@ -0,0 +1,54 @@
+# Reset TechPass password
+
+If you are a vendor and if your TechPass domain is ```techpass.gov.sg```, you may reset your TechPass password according to the [Azure Password Policy](https://learn.microsoft.com/en-us/azure/active-directory/authentication/concept-sspr-policy#administrator-password-policy-differences) or if your TechPass account gets locked.
```

Comment:
For public officers, we have the note below (i.e. >**Notes for public officers**)

For vendors, do we want to do the same so it is consistent?

Reply:
@JLeeChan , you mean to add a note for vendors?
 This whole section is applicable only for vendors and in case PO wonder how to reset their TP pwd, we direct them to reset their GSIB pwd, hence this note

Reply:
i see, then can ignore my comment